generative_data_prep/data_prep/pipeline.py

```diff
@@ -458,8 +462,8 @@ def pipeline_main(  # noqa: C901
 
     # Case 4: Do not shuffle, split file without linux OS
     elif shuffle == "False" and "linux" not in platform.lower():
-        print(SEP_STR)
-        print("You did not specify the --shuffle flag, so no shuffling was done!")
+        log_sep_str()
+        logger.warning("WARNING: you did not specify the --shuffle flag, so no shuffling was done!")
```

Comment:
does the logger not automatically add a "Warning" message in front?   Or is there some way to configure this without having to repeatedly add this string to every message?

Reply:
True. we don't need to add `WARNING` here; logger level already defines this as warning.